### ISSUE TITLE
fcn32.py "value error"

### ISSUE THREAD

**Issue by jinhang** [wkentaro/fcn#52]
hi,when I try to change pascal.py 
` target_names = np.array([
        'background',
        'newtarget',
    ])`
and I reference my VOC2012's segmentation，I prepare some png like this,but can not have the correct result.

**Comment by jinhang** [comment]
could you give me some advice?

**Comment by wkentaro** [comment]
I think debuging with iPython is the only one solution.
Please check the network input is in same format as that of original pascal.py.

**Comment by jinhang** [comment]
thanks a lot ,it successfully work,but could you give me some advices how to  enhance accuracy and velocity.

**Comment by wkentaro** [comment]
Sorry for late reply.
I think
for velocity, changing learning_rate is a method,
for accuracy, creating larger dataset, making the FCN model deeper, and integrating with other models are possible.

**Comment by wkentaro** [comment]
Please reopen when you need to do.
